dali/python/nvidia/dali/pipeline.py

```diff
@@ -116,13 +118,29 @@ class Pipeline(object):
 `enable_memory_stats`: bool, optional, default = False
     If DALI should print operator output buffer statistics.
     Usefull for `bytes_per_sample_hint` operator parameter.
+`py_num_workers`: int, optional, default = None
+    Specify the number of Python workers that will process ExternalSource callbacks.
+    If a number is not specified, the pool will use the cpu count as the number of Python workers.
```

Comment:
I don't like the idea of `cpu count` as the default. It will use all available CPU resources when multiple pipelines un in parallel.

Reply:
I set it to 1 by default now, you still need to have a parallel ExternalSource to start it.